### PR TITLE
Quick Look Video on How to use the new timetable in Timetable Administration doesn't display content ....

### PR DESCRIPTION
**Pull request by bp323**
Quick Look Video on "How to use the new timetable" in Timetable Administration app doesn't display content in Browsers 
-Internet Explorer-Version (11,10,9,8)
-Firefox- 25
-Safari-5.1.7

Operation System-Windows 7 Professional 64/32 bit

It work fine in Chrome browser.

### CODE REVIEW

**Comment by bp323** [comment]
I've made all of the required changes except for the actual video ID. I'm waiting for @mg666 to get back to us with the details of the new Youtube account.

**Comment by bp323** [comment]
Assigning to @Coenego for a round of review but this shouldn't be merged until the proper ID has been put in.

File: shared/gh/js/views/tenant-admin/gh.video.js [hunk 1]
```diff
@@ -21,26 +21,29 @@ define([], function() {
...
      *
      * @private
      */
-    var playVideo = function() {
-        $('.gh-video').addClass('isPlaying');
-        var $video = $('#gh-video').get(0);
-        $video.load();
-        $video.play();
+    var renderAndPlayVideo = function() {
+        var youtubeId = 'M7lc1UVf-VE';
```
Comment (Coenego) [inline shared/gh/js/views/tenant-admin/gh.video.js]: Would it be good if we make this configurable?

Comment (bp323) [inline shared/gh/js/views/tenant-admin/gh.video.js]: Maybe we could add it to the config file?

**Comment by bp323** [comment]
Went the direction of constants as we might be adding in more videos over time and it seems like the cleaner approach. Over to @Coenego for review.

**Pull request merged**